moped-editor/src/views/projects/projectView/ProjectPhases.js

```diff
@@ -315,6 +327,11 @@ const ProjectPhases = ({
             // extract phase_id from moped_phase object
             updatedPhasePayload.phase_id = moped_phase.phase_id;
 
+
+            // Remove extraneous fields given by MaterialTable that
+            // Hasura doesn't need
+            delete updatedPhasePayload.tableData;
```

Comment:
we do this in some other tables, but this is the first time this table complained about extraneous fields in the mutation